### PR TITLE
[Performance](sink) SIMD the tablet sink valied data function

### PR DESCRIPTION
**Pull request by HappenLee**
## Proposed changes

Before：4.5s
after：1s

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by dataroaring — APPROVED** [review]
LGTM

**Comment by dataroaring** [comment]
run buildall

**Review by zclllyybb — APPROVED** [review]
LGTM

File: be/src/vec/sink/vtablet_block_convertor.h [hunk 1]
```diff
@@ -66,15 +66,15 @@ class OlapTableBlockConvertor {
 
     Status _validate_column(RuntimeState* state, const TypeDescriptor& type, bool is_nullable,
                             vectorized::ColumnPtr column, size_t slot_index, bool* stop_processing,
-                            fmt::memory_buffer& error_prefix,
+                            fmt::memory_buffer& error_prefix, const uint32_t row_count,
```
[the remark this answers is not in the document]
Comment (kaijchen) [reply be/src/vec/sink/vtablet_block_convertor.h:69]: Remove this `const` in header file.

File: be/src/vec/sink/vtablet_block_convertor.h [hunk 2]
```diff
@@ -66,15 +66,15 @@ class OlapTableBlockConvertor {
...
                             vectorized::IColumn::Permutation* rows = nullptr);
 
     // make input data valid for OLAP table
     // return number of invalid/filtered rows.
     // invalid row number is set in Bitmap
     // set stop_processing if we want to stop the whole process now.
-    Status _validate_data(RuntimeState* state, vectorized::Block* block, int64_t& filtered_rows,
-                          bool* stop_processing);
+    Status _validate_data(RuntimeState* state, vectorized::Block* block, const uint32_t rows,
```
[the remark this answers is not in the document]
Comment (kaijchen) [reply be/src/vec/sink/vtablet_block_convertor.h:76]: Remove this `const` in header file.

**Pull request merged**